cdk/src/open-data-platform/data-plane/data-import/data-import-stack.ts

```diff
@@ -46,7 +46,7 @@ export class DataImportStack extends Construct {
 
     const { cluster, credentialsSecret } = props;
 
-    const s3BucketWithDataFiles = new s3.Bucket(this, 'open_data_platform_static_files');
+    const s3BucketWithDataFiles = new s3.Bucket(this, `${id}-open_data_platform_static_files`);
```

Comment:
@justinebreuch not sure if we want to add this permanently but left for now -- I'll remove if you prefer

Reply:
No this is great thanks!